### ISSUE TITLE
Eager API symbols not exported in the CI libtensorflow.so

### ISSUE THREAD

**Issue by eaplatanios** [tensorflow/tensorflow#12784]
The eager API symbols (i.e., `TFE_*`) do not seem to be exported in the `libtensorflow.so` built by the CI system. I tried the nightly builds and they're missing from them. I checked using `nm -D libtensorflow.so` as well as `readelf -Ws libtensorflow.so` and `objdump -TC libtensorflow.so`. I'm not sure where to look to fix this though as the Mac nightly build seems fine. @asimshankar @alextp

**Comment by asimshankar** [comment]
Note that the eager APIs are NOT stable, so use at your own risk. They may change at any time (as we're experimenting with them) and we do not plan to restrict even trivial changes (like changing symbol names) to maintain compatibility. 

With that disclaimer on the way, this is likely because [`version_script.lds`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/c/version_script.lds) prunes out all the `TFE_*` symbols. You are welcome to send in a PR :)

**Comment by eaplatanios** [comment]
Thanks for pointing me to the file. I also made a couple small changes to two other files that I found may be relevant and submitted a PR (#12785).

Thanks for the disclaimer! I'm aware of that and I'm actually trying to find out what may be broken in the API. Thanks for releasing it early in any case. It's super useful for languages that do not have anything like numpy available. It certainly helps a lot with my Scala API.
